OQ_Toolkit/OQ_Buttons/OQ_ButtonPressable.tscn

```diff
@@ -0,0 +1,24 @@
+[gd_scene load_steps=5 format=2]
+
+[ext_resource path="res://scripts/OQ_ButtonPressable.gd" type="Script" id=2]
+[ext_resource path="res://materials/bright_metal.tres" type="Material" id=4]
```

Comment:
The referenced material is not part of this commit.  Which I think is good to not have too many resources as part of the features.